cmd/config.go

```diff
@@ -178,11 +178,9 @@ func (t *TLSConfig) Load() (*tls.Config, error) {
 		ClientCAs:    rootCAs,
 		ClientAuth:   tls.RequireAndVerifyClientCert,
 		Certificates: []tls.Certificate{cert},
-		// Set the only acceptable TLS version to 1.2 and the only acceptable cipher suite
-		// to ECDHE-RSA-CHACHA20-POLY1305.
-		MinVersion:   tls.VersionTLS12,
-		MaxVersion:   tls.VersionTLS12,
-		CipherSuites: []uint16{tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305},
+		// Set the only acceptable TLS version to 1.3.
+		MinVersion: tls.VersionTLS13,
+		MaxVersion: tls.VersionTLS13,
```

Comment:
Because this TLSConfig is used for both clients and servers, switching to TLS1.3-only violates our deployability guidelines: an old RA using TLS1.2-only would be unable to talk to a new SA using TLS1.3-only. Let's keep the MinVersion at 1.2 for now.

Reply:
This is a good catch. I'll ship another commit to clamp minimum at 1.2 and restore the cipher suites.